back/spec/lib/participation_method/native_survey_spec.rb

```diff
@@ -20,6 +20,70 @@
     end
   end
 
+  describe '#create_default_form!' do
+    let(:multiloc_service) { instance_double MultilocService }
+
+    before do
```

Comment:
What I would find a more elegant way to stub translation files is to use the yml files in the spec folders (in fixtures/locales).

Reply:
I did not know that was the procedure. It is much more elegant and cleaner. Thanks for the tip.
Done.